cllc-public-app/ClientApp/src/app/components/police-representative/police-summary/police-summary.component.html

```diff
@@ -126,20 +138,19 @@ <h4>Event date</h4>
 
         <div class="d-flex">
             <label>Liquor Quantities</label>
-            <table class="servigns">
+            <table class="servings">
                 <tr>
                     <th>Type</th>
                     <th>Servings</th>
                 </tr>
-                <tr *ngFor="let item of [{type: 'draught beer', servings: 113},{type: 'wine', servings: 10}]">
-                    <td>{{item.type}}</td>
-                    <td>{{item.servings}}</td>
+                <tr *ngFor="let item of application.drinksSalesForecasts">
```

Comment:
Note that @tgunderson  is currently making a change that might make this field obsolete. Please touch base with him on this.